charts/radar-self-enrolment-ui/values.yaml

```diff
@@ -226,21 +224,40 @@ networkpolicy:
         protocol: TCP
 
 # -- Set this to ORY Kratos's Admin URL
-kratosAdminUrl: "kratos-admin"
+kratosAdminUrl: "http://kratos-admin:80/admin"
 
 # -- Set this to ORY Kratos's public URL
-kratosPublicUrl: "https://localhost/kratos"
+kratosPublicUrl: "http://kratos-public:80"
```

Comment:
I assume this would be changed to FQDN in radar-kuberetes? Not sure though as there is a browserUrl as well

Reply:
Oh this one will just use the internal URL since only the backend will be using this.